frontend/src/hooks/queryBuilder/useGetWidgetQueryRange.ts

```diff
@@ -31,7 +31,7 @@ export const useGetWidgetQueryRange = (
 			graphType,
 			selectedTime,
 			globalSelectedInterval,
-			query: stagedQuery || initialQueriesMap.metrics,
+			query: stagedQuery || initialQuery,
```

Comment:
can't we make changes in `initialQueriesMap `

Reply:
@palashgdev updated